data/xml/2023.insights.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version='1.0' encoding='UTF-8'?>
 <collection id="2023.insights">
-  <volume id="1" ingest-date="2023-04-29" type="proceedings">
+  <volume id="1" ingest-date="2023-12-21" type="proceedings">
     <meta>
-      <booktitle>The Fourth Workshop on Insights from Negative Results in NLP</booktitle>
+      <booktitle>Workshop on Insights from Negative Results in NLP</booktitle>
```

Comment:
This should be "Proceedings of the Fourth..." (see [here](https://preview.aclanthology.org/insights-reingestion/venues/insights/)).

![Uploading image.png…]()

Reply:
Done.